sentry-android-core/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,12 @@
       android:authorities="${applicationId}.SentryInitProvider"
       android:exported="false"/>
 
+    <provider
```

Comment:
`m`: I'm wondering if we could consolidate this under `SentryPerformanceProvider`? Usually introducing new content providers will add to init time, so if we can avoid introducing one, that'd be great

Reply:
sure, we can do it